src/Apps/Notifications/NotificationsApp.tsx

```diff
@@ -1,27 +1,68 @@
-import { FullBleed } from "@artsy/palette"
 import { MetaTags } from "Components/MetaTags"
 import { Notifications } from "Components/Notifications/Notifications"
 import { createFragmentContainer, graphql } from "react-relay"
 import { NotificationsApp_me$data } from "__generated__/NotificationsApp_me.graphql"
+import { Media } from "Utils/Responsive"
+import { NotificationsContextProvider } from "Components/Notifications/useNotificationsContext"
+import { NotificationQueryRenderer } from "Components/Notifications/Notification"
+import { useRouter } from "found"
+import { GridColumns, Column, Flex } from "@artsy/palette"
+import { DESKTOP_NAV_BAR_HEIGHT } from "Components/NavBar/constants"
+
+const DESKTOP_HEIGHT = `calc(100vh - ${DESKTOP_NAV_BAR_HEIGHT}px)`
+const MIN_LIST_WIDTH = 340
 
 interface NotificationsAppProps {
   me: NotificationsApp_me$data
 }
 
-const NotificationsApp: React.FC<NotificationsAppProps> = props => {
-  const { me } = props
+const NotificationsApp: React.FC<NotificationsAppProps> = ({ me }) => {
+  const { match } = useRouter()
+
+  const isNotificationsPage = match.location.pathname === "/notifications"
```

Comment:
**question:** Would a trailing slash in the URL (`/notifications/`) affect this, or would the router strip it out?

Reply:
Good catch! It does not. Replacing `===` with `startsWith` should solve the issue.